sdk/databasewatcher/arm-databasewatcher/review/arm-databasewatcher.api.md

```diff
@@ -0,0 +1,562 @@
+    apiVersion?: string;
+}
+
+// @public
+export type DatabaseWatcherProvisioningState = string;
+
+// @public
+export interface Datastore {
+    adxClusterResourceId?: string;
+    kustoClusterDisplayName?: string;
+    kustoClusterUri: string;
+    kustoDatabaseName: string;
+    kustoDataIngestionUri: string;
+    kustoManagementUrl: string;
+    kustoOfferingType: KustoOfferingType;
+}
+
+// @public
+export interface ErrorAdditionalInfo {
+    readonly info?: Record<string, any>;
+    readonly type?: string;
+}
+
+// @public
+export interface ErrorDetail {
+    readonly additionalInfo?: ErrorAdditionalInfo[];
+    readonly code?: string;
+    readonly details?: ErrorDetail[];
+    readonly message?: string;
+    readonly target?: string;
+}
+
+// @public
+export interface ErrorResponse {
+    error?: ErrorDetail;
+}
+
+// @public
+export interface HealthValidation extends ProxyResource {
+    properties?: HealthValidationProperties;
+}
+
+// @public
+export interface HealthValidationProperties {
+    readonly endTime: Date;
+    readonly issues: ValidationIssue[];
+    readonly provisioningState?: ResourceProvisioningState;
+    readonly startTime: Date;
+    readonly status: ValidationStatus;
+}
+
+// @public
+export interface HealthValidationsGetOptionalParams extends OperationOptions {
+}
+
+// @public
+export interface HealthValidationsListByParentOptionalParams extends OperationOptions {
+}
+
+// @public
+export interface HealthValidationsOperations {
+    get: (resourceGroupName: string, watcherName: string, healthValidationName: string, options?: HealthValidationsGetOptionalParams) => Promise<HealthValidation>;
+    listByParent: (resourceGroupName: string, watcherName: string, options?: HealthValidationsListByParentOptionalParams) => PagedAsyncIterableIterator<HealthValidation>;
+    startValidation: (resourceGroupName: string, watcherName: string, healthValidationName: string, options?: HealthValidationsStartValidationOptionalParams) => PollerLike<OperationState<HealthValidation>, HealthValidation>;
+}
+
+// @public
+export interface HealthValidationsStartValidationOptionalParams extends OperationOptions {
+    updateIntervalInMs?: number;
+}
+
+// @public
+export enum KnownActionType {
+    Internal = "Internal"
+}
+
+// @public
+export enum KnownAlertRuleCreationProperties {
+    CreatedWithActionGroup = "CreatedWithActionGroup",
+    None = "None"
+}
+
+// @public
+export enum KnownCreatedByType {
+    Application = "Application",
+    Key = "Key",
+    ManagedIdentity = "ManagedIdentity",
+    User = "User"
+}
+
+// @public
+export enum KnownDatabaseWatcherProvisioningState {
+    Canceled = "Canceled",
+    Failed = "Failed",
+    Succeeded = "Succeeded"
+}
+
+// @public
+export enum KnownKustoOfferingType {
+    Adx = "adx",
+    Fabric = "fabric",
+    Free = "free"
+}
+
+// @public
+export enum KnownManagedServiceIdentityType {
+    None = "None",
+    SystemAndUserAssigned = "SystemAssigned, UserAssigned",
```

Comment:
is this original name? typespec link?

Reply:
https://github.com/Azure/typespec-azure/blob/main/packages/typespec-azure-resource-manager/lib/Legacy/managed-identity.tsp#L60-L73